### DISCUSSION TITLE
Cyberduck crashes when trying to connect

### DISCUSSION THREAD

**Discussion by pdelaney27** [iterate-ch/cyberduck#14961] [Q&A]
When launching Cyberduck, the application crashes immediately

**Comment by platypusrex256** [comment]
I have this same issue. Seems hella buggy and support here is zilch!

**Comment by amster2000** [comment]
Same. Tested on 2 Mac devices, same result. Reinstall did not fix the issue. A new connection needs to be set up each time, which is frustrating to say the least.

**Reply by platypusrex256** [reply]
App isn't actually crashing for me... just hella lag. I give it some space and it chugs along but I'm really not impressed.

**Reply by platypusrex256** [reply]
Okay this might help for you... but if you can narrow your path in your connection to exactly what you want to mount, it might help with the crashing. I had an issue because my Google Drive connection was 50TB and it takes forever for Finder to index all that stuff. So I narrowed it down to exactly what I need (2TB) and it did a much better job. Still feels buggy though!

**Comment by tatze96** [comment]
@pdelaney27 Please add the latest [crash report](https://docs.cyberduck.io/cyberduck/support/#crash-reports) for the mentioned crashes on startup.

*Sidenote*: Ensure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

@platypusrex256 Are you using Mountain Duck or Cyberduck? Based on your most recent comment, you likely use Mountain Duck as Finder doesn't have access to the file listing within Cyberduck. If you're indeed using Mountain Duck, this is a completely different issue as the initial person referred to Cyberduck.

@amster2000 Do you refer to crashes of Cyberduck while connecting to a bookmark?

**Comment by Kreatifart** [comment]
Mountainduck crashes after a while after synchronization starts... 
I did add the latest crash report for the mentioned crashes on startup.

Mac mini - 2023
Apple M2
macOS 14.2 Beta (23C5055b)

[Mountain Duck-2023-12-03-231226.zip](https://github.com/iterate-ch/cyberduck/files/13540122/Mountain.Duck-2023-12-03-231226.zip)

**Reply by tatze96** [reply]
Those crashes might already be resolved within the current *Snapshot* version.

You can update to the *Snapshot* version by choosing *Snapshot Builds* instead of *Release* within the Mountain Duck preferences *Update* tab and clicking on the *Check for Update Now* button afterward.

You can set the update setting back to *Release* if you want to revert to the release updates as soon as a new release version is available.
